public/js/selfoss-base.js

```diff
@@ -621,6 +621,47 @@ var selfoss = {
             });
         });
         selfoss.logout();
+    },
+
+
+    /**
+     * Copy a string to the users clipboard.
+     * This will deselect anything the user has selected and overwirte their clipboard.
+     *
+     * @param stringToCopy
+     * @return true if no execptions were thrown
+     */
+    copyString: function(stringToCopy) {
```

Comment:
Note, once we merge #1137, we will be able to use something like https://www.npmjs.com/package/copy-to-clipboard.

Reply:
Thoughts on merging #1137 into this branch then blocking it until #1137 has been merged?

Reply:
👍Hopefully, it will not take much longer to merge it.

Reply:
Yay, it is in master now.